config/default/kustomization.yaml

```diff
@@ -18,11 +18,11 @@ bases:
 - ../manager
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
 # crd/kustomization.yaml
-#- ../webhook
+- ../webhook
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
-#- ../certmanager
+- ../certmanager
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
-- ../prometheus
```

Comment:
Why comments this line? If it's not related to the webhook, please leave it as uncomment for the prometheus usage.

Reply:
Good point, prometheus is not needed.

Reply:
Prometheus is back in, resolving this conversation.